#block
int32 blocked_size
int32 lidar_zero_count
bool blocked_judge

#restart
bool lidar_trap_restart
